src/main/java/gov/nasa/cumulus/metadata/aggregator/MetadataFilesToEcho.java

```diff
@@ -251,6 +269,10 @@ public void setGranuleFileSizeAndChecksum(JSONArray input_granules) {
 		AdapterLogger.LogInfo(this.className + " GranuleArchive HashSet Size:" + granule.getGranuleArchiveSet().size());
 		AdapterLogger.LogInfo(this.className + " GranuleArchive HashSet:" + granule.getGranuleArchiveSet());
 	}
+	public void readIsoMetadataFile(String file, String s3Location, String collection_short_name)
```

Comment:
Where is this being used?

Reply:
First of all appreciate everybody's reviews/inputs
all import * have been cleaned up.  Unused code removed and Logtitude logic fixed.

According to handling the ISO parsing exception.  I made some small changes in MetadataFilesToEcho.java 's readIsoMetadataFile function

*  While exception found during parseRequiredFields, we log exception and re-throw
*  When getting into more specific processing , i.e. isoType == IsoType.MENDS or isoType == IsoType.SMAP, some of the functions being called by readIsoMetadataFile are throwing "forced exceptions", so catch a generic exception at line 323 get them all plus the un-expected exception.  Log and re-throw.  this should help us with debugging for throwing the exception will finally print the exception stack and allow us to identify the line of problem.

I disagree with the previous code which catch the XPathExpressionException , log , swallow and go on.  I think if we have XPathExpressionException, we should still throw an exception.

Please feel free to address further comments/suggestions.